Makefile

```diff
@@ -1,4 +1,11 @@
-CACHITO_COMPOSE_ENGINE ?= docker-compose
+DOCKER_COMPOSE_V2_AVAILABLE := $(shell docker compose version >/dev/null 2>&1 && echo true || echo false)
+
+ifeq ($(DOCKER_COMPOSE_V2_AVAILABLE), true)
+	CACHITO_COMPOSE_ENGINE ?= docker compose
+else
+	CACHITO_COMPOSE_ENGINE ?= docker-compose
+endif
+
```

Comment:
If v1 has been deprecated, why can't we just get rid of it completely? Why do we want to fall back to it?

Reply:
I mean any recentish distro should have v2 already, shouldn't it?

>  I've always used podman-compose, so I really don't care either way

[off-topic] good for you, podman-compose either isn't available or sucks big time on SoC Debian ARM spins :(

Reply:
Oops, I forgot to respond here that I switched it to v2